integration_tests/pom.xml

```diff
@@ -107,7 +107,7 @@
         </dependency>
         <dependency>
             <groupId>com.nvidia</groupId>
-            <artifactId>rapids-4-spark-udf-examples</artifactId>
+            <artifactId>rapids-4-spark-udf-examples_${scala.binary.version}</artifactId>
```

Comment:
did we have an issue to update the rest of IT builds?

Reply:
I tried to catch all the builds that are in this repo.  Just filed #1640 to track build scripts in other repos.